### PR TITLE
Profiles index page

### PR DESCRIPTION
**Pull request by jonmauney**
/domain @noveng05 @agirlnamedsophia @kathleen @alanjenson1 

/task https://app.asana.com/0/346606283875063/358313893620834

Add a profiles index page. Next we will make it searchable. 
screen shot 2017-06-06 at 7 45 05 pm

### CODE REVIEW

**Comment by jonmauney** [comment]
@noveng05 status update?

**Comment by jonmauney** [comment]
bump

File: app/views/profiles/_search_result.html.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div>
+  <%= image_tag profile.picture_url, size: 32 %>
+  <%= link_to profile.full_name, profile_path(profile.id) %>
+</div>
```
Comment (agirlnamedsophia) [inline app/views/profiles/_search_result.html.erb:4]: Would it make more sense to call this partial "_profile.html.erb"?

**Comment by agirlnamedsophia** [comment]
just one question about the naming!

<< domain LGTM

File: app/views/profiles/_search_result.html.erb (hunk 1 above)
Comment (jonmauney) [reply app/views/profiles/_search_result.html.erb:4]: this naming is for adding search functionality later (not just for profiles)

**Pull request merged**